thunder/recipes/base.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         plugins=None,
     ):
         super().__init__(interpreter=interpreter, plugins=plugins)
-        self.executor_names = ["cudnn", "sdpa", "torchcompile_xentropy"]
+        self.executor_names = ["sdpa", "torchcompile_xentropy"]
         self.fuser = fuser
```

Comment:
I think we might want to keep cudnn required by default (and see that it is installed if we don't have it currently).